components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -35,7 +35,7 @@
                             "public"
                         ],
                         "default": "private",
-                        "description": "Whether the port visibility should be private or public. 'private' (default) will only allow users with workspace access to access the port. 'public' will allow everyone with the port URL to access the port."
+                        "description": "Whether the port visibility should be private or public. 'private' (default) will only allow users with workspace access to access the port. 'public' will allow everyone with the port URL to access the port, can help with CORS issues."
```

Comment:
Let's drop this change. It's feels too random here.

Would be great if we had a docs page for this, and list the differences in behavior. :cloud:

Reply:
> Would be great if we had a docs page for this, and list the differences in behavior. ☁️

Maybe you mean something like this?
- https://github.com/gitpod-io/website/pull/2950

cc @loujaybee 👀